spec.html

```diff
@@ -41537,6 +41592,28 @@ <h1>
           <p>The parameter _iterable_ is expected to be an object that implements an @@iterator method that returns an iterator object that produces a two element array-like object whose first element is a value that will be used as a Map key and whose second element is the value to associate with that key.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-add-value-to-keyed-group" type="abstract operation">
+        <h1>
+          AddValueToKeyedGroup (
+            _groups_: a List of Records that have [[Key]] and [[Elements]] fields,
+            _key_: an ECMAScript language value,
+            _value_: an ECMAScript language value,
+          ): ~unused~
+        </h1>
+        <dl class="header">
+        </dl>
+        <emu-alg>
+          1. For each Record { [[Key]], [[Elements]] } _g_ of _groups_, do
+            1. If SameValue(_g_.[[Key]], _key_) is *true*, then
+              1. Assert: Exactly one element of _groups_ meets this criteria.
```

Comment:
```suggestion
              1. Assert: Exactly one element of _groups_ meets this criterion.
```

Reply:
criteria is both singular and plural, but sure

Reply:
> criteria is both singular and plural

Huh. Hadn't heard that.